io/src/ply_io.cpp

```diff
@@ -39,6 +39,7 @@
 #include <pcl/common/io.h>
 #include <pcl/io/ply_io.h>
 
+#include <algorithm>
```

Comment:
Please undo all remaining changes in this file (whitespace changes and unnecessary `<algorithm>` include)